src/popup/router/components/InviteItem.vue

```diff
@@ -1,36 +1,58 @@
 <template>
   <div class="invite-row">
     <div class="invite-info">
-      <span class="primary">{{ balance }} {{ $t('pages.appVUE.aeid') }}</span>
+      <span class="primary">
+        <span class="balance">{{ balance }}</span>
+      </span>
       <!--eslint-disable-line vue-i18n/no-raw-text-->
-      ({{ (balance * current.currencyRate).toFixed(2) }}
+      ({{ currencySigns[current.currency.toUpperCase()] }}
       <!--eslint-disable-next-line vue-i18n/no-raw-text-->
-      {{ current.currency.toUpperCase() }})
+      {{ (balance * current.currencyRate).toFixed(2) }})
       <span class="date">{{ createdAt | formatDate }}</span>
     </div>
     <div class="invite-link">
       <span>{{ link }}</span>
       <button class="invite-link-copy" v-clipboard:copy="link"><CopyIcon /></button>
     </div>
     <template v-if="!topUp">
-      <Button half dark @click="topUp = true">{{ $t('pages.invite.top-up') }}</Button>
-      <Button half @click="claim">{{ $t('pages.invite.claim') }}</Button>
+      <div class="center">
+        <Button v-if="balance > 0" class="minwidth" bold inline @click="claim">{{
+          $t('pages.invite.claim')
+        }}</Button>
+        <Button v-else class="minwidth" bold inline dark @click="deleteItem">{{
+          $t('pages.invite.delete')
+        }}</Button>
+        <Button class="minwidth" bold inline :disabled="!(tokenBalance > topUpAmount)" @click="topUp = true">{{
+          $t('pages.invite.top-up')
+        }}</Button>
+      </div>
     </template>
     <template v-else>
-      <AmountSend v-model="topUpAmount" />
-      <Button half dark @click="topUp = false">{{ $t('pages.invite.close') }}</Button>
-      <Button half @click="sendTopUp">{{ $t('pages.invite.top-up') }}</Button>
+      <AmountSend
+        class="send-amount"
```

Comment:
```suggestion
```
I propose to use `amount-send-container` for now and later refactor it to `amount-send`. I think this convention is better than adding extra classes for additional styles.